Banking.Console/Program.cs

```diff
@@ -1,2 +1,68 @@
-﻿// See https://aka.ms/new-console-template for more information
-Console.WriteLine("Hello, World!");
+﻿int AccountId = 0;
+int AccountBalance = 0;

```

Comment:
Remember the following rules in C# naming convention:
- variables in methods start with small
- private variables in classes starts with _ and then small char. (_accountId)
- Properties always start with upper case letter
- public variables in classes also called fields start with small